l10n/DEFAULT/Moose_10_05_21.lua

```diff
@@ -8882,7 +8882,7 @@ end
 -- @field #SETTINGS
 SETTINGS = {
   ClassName = "SETTINGS",
-  ShowPlayerMenu = true,
+  ShowPlayerMenu = false,
```

Comment:
What does this do?

Reply:
Part of the settings removal thing. Its a two parter, this and the CavCaller file.